### PR TITLE
chore(*): design changes and consistency

### PR DESCRIPTION
**Pull request by almostSouji**
- Color enums
- Wrap user tag in inline code
- Raidnuke: Log: Discern dry run in color and wording (transparent if dry, red if actual)
- Raidnuke: verbose option names
- Raidnuke: consistent, centralized attachment output
- Raidnuke: consistent, centralized core handling of all 3 variants
- Raidnuke: translate hardcoded strings
- Raidnuke: centralized time range validation
- Log: Thread create/delete consistent color (discord magenta)

> **Note**
> requires a command deploy

### CODE REVIEW

**Comment by almostSouji** [comment]
> **Warning**
> Cannot test file upload at the moment, due to upstream issues (Discord)

File: packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts [hunk 1]
```diff
@@ -0,0 +1,213 @@
+import dayjs from 'dayjs';
+import {
+	ButtonInteraction,
+	ButtonStyle,
+	ChatInputCommandInteraction,
+	Collection,
+	ComponentType,
+	GuildMember,
+	ModalSubmitInteraction,
+	Snowflake,
```
Comment (iCrawl) [inline packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts:10]:
```suggestion
	type ButtonInteraction,
	ButtonStyle,
	type ChatInputCommandInteraction,
	Collection,
	ComponentType,
	type GuildMember,
	type ModalSubmitInteraction,
	type Snowflake,
```

File: packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
+
+	parameterStrings.splice(
+		0,
+		0,
+		i18next.t('command.mod.anti_raid_nuke.common.parameters.heading', { lng: locale }),
+		i18next.t('command.mod.anti_raid_nuke.common.parameters.days', {
+			count: pruneDays,
+			lng: locale,
+		}),
+	);
```
Comment (iCrawl) [inline packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts:88]: Why mutate like that 🤔

File: packages/yuudachi/locales/en-US/translation.json [hunk 3]
```diff
@@ -459,7 +459,7 @@
 			"anti_raid_nuke": {
 				"title": "Anti-Raid-Nuke launched",
 				"title_dryrun": "Anti-Raid-Nuke dry run",
-				"description": "• Author: {{- author}}\n• Hit count: {{count}}"
+				"description": "• Moderator: {{- author}}\n• Hit count: {{count}}"
```
Comment (iCrawl) [inline packages/yuudachi/locales/en-US/translation.json:462]: Might as well rename the arg too

File: packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts (hunk 2 above)
Comment (almostSouji) [reply packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts:88]: looks absolutely horrible, now that i look at it again

File: packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+	} else if (collectedInteraction?.customId === banKey || collectedInteraction?.customId === dryRunKey) {
+		const dryRunMode = collectedInteraction.customId === dryRunKey;
+
+		const content =
+			collectedInteraction.message.content +
+			(dryRunMode ? `\n\n${i18next.t('command.mod.anti_raid_nuke.common.parameters.dry_run', { lng: locale })}` : '');
+
+		await collectedInteraction.update({
+			content,
+			components: [
```
[the remark this answers is not in the document]
Comment (almostSouji) [reply packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts:162]: i suppose right now it makes no sense to have buttons at this stage, should be re-visited with progress updates and panic button, if that ever happens

[the remark this answers is not in the document]
Comment (JPBM135) [reply packages/yuudachi/src/commands/moderation/sub/anti-raid-nuke/coreCommand.ts]: Agreed

**Pull request merged**